Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -6,11 +6,14 @@ class FormTextField: UIView {
     // MARK: - Outlets
     //
     
-    @IBOutlet weak private(set) var contentView: UIView!
```

Comment:
Why changed from `private(set)` to `private`? 
How will you test the outlets if they are private?

Reply:
even don't access the outlets from view controller just working with available functions.

Reply:
Yes but we need to access them for testing.